src/main/java/seedu/address/model/person/Seller.java

```diff
@@ -22,7 +23,8 @@ public class Seller extends Person {
      * @param houses      The houses the seller has (modified to accept a list of houses)
      * @param tags        The tags associated with the seller.
      */
-    public Seller(Name name, Phone phone, Email email, String housingType, ArrayList<House> houses, Set<Tag> tags) {
+    public Seller(Name name, Phone phone, Email email, HousingType housingType, ArrayList<House> houses,
```

Comment:
Is housingType still stored under Person? If I am not wrong, the housingType is just a validation to see what kind of houses should be stored, it should not be stored under Person.

Reply:
I agree with khoon sun